test/helpers/weights.ts

```diff
@@ -0,0 +1,65 @@
+import "@moonbeam-network/api-augment";
+import { DevModeContext } from "@moonwall/cli";
+import { GLMR, KeyringPair, MIN_GLMR_STAKING, alith, generateKeyringPair } from "@moonwall/util";
+import chalk from "chalk";
+import { Debugger } from "debug";
+
+export async function createAccounts(
+  context: DevModeContext,
+  maxAccounts: number
+): Promise<KeyringPair[]> {
+  const randomAccounts = new Array(Number(maxAccounts)).fill(0).map(() => generateKeyringPair());
+
+  let alithNonce = await context
+    .viem()
+    .getTransactionCount({ address: alith.address as `0x${string}` });
+  await context.createBlock(
+    randomAccounts.map((randomCandidate) =>
+      context
+        .polkadotJs()
+        .tx.sudo.sudo(
+          context
+            .polkadotJs()
+            .tx.balances.forceSetBalance(
+              randomCandidate.address,
+              12n * MIN_GLMR_STAKING + 50n * GLMR
+            )
+        )
+        .signAsync(alith, { nonce: alithNonce++ })
+    ),
+    { allowFailures: false }
+  );
+
+  return randomAccounts;
+}
+
+export async function countExtrinsics(
+  context: DevModeContext,
+  method: string,
+  logger: Debugger
```

Comment:
Do we want to move to having the logger by passed to functions ? That seems like a big change